Marble Blast Platinum/platinum/client/scripts/loadinggui.cs

```diff
@@ -318,46 +319,58 @@
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Sending us a bug report is the best way to get it fixed! Let us know about any issues on our Discord server or the website's forums.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Sending us a bug report is the best way to get it fixed! Let us know about any issues by creating a thread in #bug-report in our official Discord server, or at Marble Blast Discussion, Support, and Fun > Marble Blast Support on our website's forums.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Halloween and Xmas themed level remakes often contain new achievements and flairs for you to attain. Join us for these events!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Craving Mega Marble Wars? A unique Mega Marble mode gives you points based on how hard you Mega Marble your opponents. Can you be the King?";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: When creating a recording, it will automatically restart itself until you beat the level or exit it. You can always replay the level and a new recording will be made.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you get a World Record, the game will automatically upload a recording to the server, save one locally and announce your achievement to our Discord server.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you get a World Record, the game will automatically upload a recording to the server, save one locally, and announce your achievement in #speedrunning on our official Discord server.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you fall Out of Bounds, press the powerup key to respawn faster! In Hunt levels, if you're carrying a powerup, it will not be consumed.";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: If you press the Respawn button in Multiplayer, it will de-activate for 25 seconds. No cheating to get closer to the gems!";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Besides filling the Blast Bar to its maximum power, the Blast Item gives you a small 3% boost!";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Besides filling the Blast Meter to its maximum power, the Blast Item gives you a small 3% boost!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Mega Marble powerup will make your opponents run in fear! If two Mega Marbles collide, the slower one will get booted away!";
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Mega Marbles are very useful for collecting loads of tightly-packed Gems.";
-$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Fireball powerup takes priority over the standard multiplayer Blast. If you immediately activate Blast after using Fire Blast, then your boost will be much bigger.";
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: The Fireball powerup takes priority over the standard Blast. If you immediately activate Blast after using Fire Blast, then your boost will be much bigger.";
```

Comment:
Maybe it's rare but it's possible that there are singleplayer levels with both blast enabled and fireball powerups, so this isn't actually unique to multiplayer. Can probably be reverted

Reply:
Are you colorblind perchance

Reply:
Are you colorblind perchance

Reply:
LOL gotcha